static/js/course_detail.js

```diff
@@ -8,6 +8,12 @@ $(document).ready(function() {
     $("li.extra-faq").toggleClass("d-none");
     $readmoreText.text($readmoreText.text() === "Hide" ? "Read" : "Hide");
   });
+
+  $(".dates-tooltip").popover({
+    template:
+      '<div class="popover" role="tooltip"><div class="arrow"></div><div class="popover-header px-0 mx-5"></div><div class="popover-body"></div></div>',
```

Comment:
can we please split into lines for readability ?

Reply:
Done.